ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/TimeServiceBackend.scala

```diff
@@ -21,6 +21,9 @@ object TimeServiceBackend {
   def simple(startTime: Instant): TimeServiceBackend =
     new SimpleTimeServiceBackend(startTime)
 
+  def wallClock(clock: Clock): TimeServiceBackend =
```

Comment:
If we change the ledgers to accept `TimeProvider`s, not `TimeServiceBackend`s, we can drop this in favor of `TimeProvider.UTC`.

Reply:
`StandaloneApiServer` requires an optional `TimeServiceBackend` though, so I don't think I can get rid of it easily. What I could do is to provide a read-only wrapper for `TimeProvider`, or to leave it as it is and have both defaults (aligned) in the sandbox ledgers.

Reply:
In the case of wall clock time, we just give it `None`. This is important because if it's a `Some(service)`, the API server will spin up time manipulation endpoints (`TimeService.GetTime` and `TimeService.SetTime`).

Reply:
Is there test coverage about that (i.e. that sandbox should not spin up manipulation endpoints with either backend)?

Reply:
Unfortunately not, but that's a good point; we could add some. Don't worry about it in this PR though.